packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -147,6 +152,14 @@
 	}
 }
 
+.block-editor-global-styles-background-panel__color-tools-panel-item {
+
+	button.components-button {
```

Comment:
While style overrides are 

Instead of using `button.components-button` (which is meant to be a private API of the components package), could we assign a custom classname to the `button` itself and use that to apply style overrides?

Also, do we need to use a `Button` component, if we're going to override its styles? Should we consider just use a vanilla `button` instead?

---

(These comments apply to all instances of style overrides for `Button` in this file)